src/ert/_c_wrappers/config/config_content.py

```diff
@@ -7,11 +7,52 @@
 from ert._c_wrappers.config.config_path_elm import ConfigPathElm
 from ert._c_wrappers.config.content_type_enum import ContentTypeEnum
 from ert._c_wrappers.config.schema_item import SchemaItem
+from ert._clib.config_keys import ConfigKeys
 
 if TYPE_CHECKING:
     from ert._c_wrappers.config.config_parser import ConfigParser
 
 
+SINGLE_VALUE_KEYS = [
+    ConfigKeys.ANALYSIS_SELECT,
+    ConfigKeys.CONFIG_DIRECTORY,
+    ConfigKeys.DATAROOT,
+    ConfigKeys.DATA_FILE,
+    ConfigKeys.ENSPATH,
+    ConfigKeys.RERUN_KEY,
+    ConfigKeys.ALPHA_KEY,
```

Comment:
I have a strong feeling this list was sorted at some point. Suggest to redo that?

Reply:
Good point, it was never sorted but it should be sorted.